include/os/freebsd/spl/sys/debug.h

```diff
@@ -74,9 +74,9 @@ void spl_dumpstack(void);
 	    "%s", "VERIFY(" #cond ") failed\n"))
 
 #define	VERIFY3B(LEFT, OP, RIGHT)	do {				\
-		boolean_t _verify3_left = (boolean_t)(LEFT);		\
-		boolean_t _verify3_right = (boolean_t)(RIGHT);		\
-		if (!(_verify3_left OP _verify3_right))			\
+		const boolean_t _verify3_left = (boolean_t)(LEFT);		\
+		const boolean_t _verify3_right = (boolean_t)(RIGHT);		\
+		if (unlikely(!(_verify3_left OP _verify3_right)))			\
```

Comment:
nit: it's hard to tell in the GitHub review interface, but let's make sure the trailing `\`'s stay aligned.

Reply:
I'm pretty sure the trailing \'s weren't aligned to begin with - at least in my editor with tabwidth=4 - but they should be now!